packages/core/src/client.ts

```diff
@@ -73,11 +76,13 @@ export const createClient = (opts: ClientOptions) => new Client(opts);
 
 /** The URQL application-wide client library. Each execute method starts a GraphQL request and returns a stream of results. */
 export class Client {
+  // Event target for monitoring
+  debugTarget?: Target;
+
   // These are variables derived from ClientOptions
   url: string;
   fetch?: typeof fetch;
   fetchOptions?: RequestInit | (() => RequestInit);
-  exchange: Exchange;
```

Comment:
Assuming this is intended to be a private variable - let me know if that isn't the case.